docs/content/docs/development/guide.md

```diff
@@ -93,6 +93,22 @@ rest.port: 8081
 rest.address: localhost
 ```
 
+### Uninstalling the operator locally
+```bash
+helm uninstall flink-operator
+```
+
+### CRD Generating and Upgrading 
```

Comment:
Should be: **Generating and Upgrading the CRD**

Reply:
Fixed